src/Nordea/Resources/Colors.elm

```diff
@@ -118,6 +119,11 @@ grayWarm =
     hex "#F4F2F1"
 
 
+hoverGray : Color
+hoverGray =
```

Comment:
The naming pattern for the other colors is to have the color first and a variant second:
```suggestion
grayHover : Color
grayHover =
```

It makes autocomplete better 🤩